### PR TITLE
Cleanup for v1

### PR DESCRIPTION
**Pull request by ebebbington**
Fixes #{issue ref}

**Description**

* deno fmt
* deno lint
* updates dependencies
* add ci workflows

**Other Notes**

* **Do not review all at once lol**, would be best to review commit by commit, excluding the deno fmt and deno lint commits

### CODE REVIEW

**Comment by ebebbington** [comment]
ehhhhh i'll fix tomorrow

**Comment by ebebbington** [comment]
ok ready for review

**Comment by ebebbington** [comment]
finally adding #63 @Guergeiro ;)

File: .docker/update_deno_version_in_dockerfile.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const fetchRes = await fetch("https://cdn.deno.land/deno/meta/versions.json");
```
Comment (crookse) [inline .docker/update_deno_version_in_dockerfile.ts:1]: we could use the bumper script now :)

Comment (ebebbington) [reply .docker/update_deno_version_in_dockerfile.ts:1]: Ah yes i will update thee ci stuff :P

**Comment by crookse** [comment]
i see the following comment sprinkled throughout:

```
// @ts-ignore Because Drash expects results from getBodyParam to be a string. An issue should probably be made for this
```

`getBodyParam()` has the following return types:

```
string | { [key: string]: unknown } | Array<unknown> | boolean | null
```

Is this an issue with the realworld app or drash?

**Comment by ebebbington** [comment]
> i see the following comment sprinkled throughout:
> 
> ```
> // @ts-ignore Because Drash expects results from getBodyParam to be a string. An issue should probably be made for this
> ```
> 
> `getBodyParam()` has the following return types:
> 
> ```
> string | { [key: string]: unknown } | Array<unknown> | boolean | null
> ```
> 
> Is this an issue with the realworld app or drash?

Ah this was from a while ago, i did make an issue and add a PR to it so those comments and the code can be addressed

**Comment by ebebbington** [comment]
@crookse https://github.com/drashland/deno-drash-realworld-example-app/pull/100/commits/ce48d93afc9514c050ef580030eae0b54ebab53f

File: .docker/update_deno_version_in_dockerfile.ts (hunk 1 above)
Comment (ebebbington) [reply .docker/update_deno_version_in_dockerfile.ts]: https://github.com/drashland/deno-drash-realworld-example-app/pull/100/commits/8c84384d3100114ed745ab4721eb2aeaed593a47

**Comment by ebebbington** [comment]
Commenting out linter because deno lint has a bug, ill make an issue for this instead of waiting for a release

**Comment by ebebbington** [comment]
I'm done @crookse, the PR is finalised

**Review by crookse — APPROVED** [review]
LGTM

**Pull request merged**